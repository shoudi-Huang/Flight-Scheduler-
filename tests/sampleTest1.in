flight add monday 18:00 Sydney Melbourne 120
location add Sydney -33.87 150.6 0.5
locations
exit

