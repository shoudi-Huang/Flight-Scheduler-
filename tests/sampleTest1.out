User: Invalid starting location.

User: Successfully added location Sydney.

User: Locations (1):
Sydney

User: Application closed.
